src/styles/blocks/header.css

```diff
@@ -0,0 +1,55 @@
+.header {
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: white;
+  width: 100%;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.header__logo {
+  margin: 10px;
+  padding-right: auto;
+  height: 40px;
+  width: 40px;
+}
+
+.header__nav {
+  display: flex;
+  flex-direction: row;
+  padding-right: 50px;
+}
+
+.header__nav__link {
+  color: #000;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
```

Comment:
I believe this should be higher. By the way the design and tests look, I would say it's either 600 or 700, not 500